src/auxiliary/mpi.jl

```diff
@@ -75,4 +69,62 @@ end
 end
 
 
-end # @muladd
+"""
+    ode_norm(u, t)
+
+Implementation of the weighted L2 norm of Hairer and Wanner used for error-based
+step size control in OrdinaryDiffEq.jl. This function is aware of MPI and uses
+global MPI communication when running in parallel.
```

Comment:
> [...] uses global MPI communcation [...]

General remark: This is a - potentially - significant downside of this considerably simplified approach: We have no control over the MPI communicator that is used. While this is clearly much easier to implement and use (and harder to use wrongly) than the approach implemented in #1104, it also leaves us with less flexibility.

Reply:
Yes, correct. However, we don't have any other kind of communication in Trixi yet, so it's not really a difference right now.

Reply:
Nope, I agree. We should keep the other PR though (or at least keep a reference to it) should we decide to go ahead with this one here.